ambry-frontend/src/main/java/com/github/ambry/frontend/PostBlobHandler.java

```diff
@@ -265,12 +265,17 @@ private BlobInfo getBlobInfoFromRequest() throws RestServiceException {
           frontendMetrics.postBlobMetricsGroup);
       BlobProperties blobProperties = RestUtils.buildBlobProperties(restRequest.getArgs());
       Container container = RestUtils.getContainerFromArgs(restRequest.getArgs());
+      Long chunkUploadTtl =
+          RestUtils.getLongHeader(restRequest.getArgs(), RestUtils.Headers.CHUNK_UPLOAD_TTL, false);
       if (blobProperties.getTimeToLiveInSeconds() + TimeUnit.MILLISECONDS.toSeconds(
           blobProperties.getCreationTimeInMs()) > Integer.MAX_VALUE) {
         LOGGER.debug("TTL set to very large value in POST request with BlobProperties {}", blobProperties);
         frontendMetrics.ttlTooLargeError.inc();
       } else if (container.isTtlRequired() && (blobProperties.getTimeToLiveInSeconds() == Utils.Infinite_Time
-          || blobProperties.getTimeToLiveInSeconds() > frontendConfig.maxAcceptableTtlSecsIfTtlRequired)) {
+          || chunkUploadTtl == null
```

Comment:
Nit: Would be good to reconcile the max_ttl first. Something like:

final long max_ttl = chunkUploadTtl != null ? frontendConfig.chunkUploadMaxChunkTtlSecs : frontendConfig.maxAcceptableTtlSecsIfTtlRequired;
.
.
if (... blobProperties.getTimeToLiveInSeconds() > max_ttl) {

}